lisa/analysis/rta.py

```diff
@@ -160,15 +162,28 @@ def get_log_file(self, task):
         """
         return self.perf_data[task]['logfile']
 
-    def get_df(self, task):
+    @memoized
```

Comment:
not sure if should try to memoize that, as this means that it will keep the dataframe and the trace alive forever in the Python process it was started. Long running notebooks may suffer from that.

Reply:
Yes, that memoized is quite controversial, it's difficult to say when it makes sense and when it doesn't.
It's likely true that if you wanna normalized data you will keep to use them for all the analysis.
But it's also true that it's not so expensive to build them on demand.... thus, I'll get rid of it! ;)